pages/blog/posts/new-website.md

```diff
@@ -0,0 +1,51 @@
+---
+title: "Announcing our new website"
+date: "2023-10-02"
+type: Community
+cover: /img/posts/2023/new-website/cover.webp
+authors:
+  - name: Benjamin Granados
+    photo: /img/avatars/benjagm.png
+    link: https://twitter.com/benjagm
+excerpt: "We are so pleased to announce the launch of our new website!"
+---
+
+## We are so pleased to announce the launch of our new website!
+
+We are excited to announce the launch of our newly designed JSON Schema website!
+
+We have created a modern design with improved site navigation and new information architecture to help you find exactly what you are looking for and better explain the benefits of JSON Schema and its incredible ecosystem.
+
+## A little bit of history
+
+The JSON Schema story started in 2007 when Chris Zyp proposed JSON Schema as a schema language to validate JSON documents. Today, JSON Schema is one of the most used standards in the industry. During all this time, there have been different leading groups, specification releases, site versions, and, most importantly, hundreds of contributions by the JSON Schema Community. However, we have been receiving feedback about different ways to improve and refresh the site experience, and this is why the Community has been working so hard to release a new version of the website to help JSON Schema continue this amazing growth!
+
+<figure className="mt-5">
+  <img class="max-w-xs sm:max-w-md md:max-w-lg lg:max-w-xl xl:max-w-2xl mx-auto" src="/img/posts/2023/new-website/site-2016.webp"/>
+    <figcaption className="mt-2 mb-10 text-sm text-center text-gray-500">The JSON Schema website back in 2016</figcaption>
+</figure>
+
+## A new story
+
+Our new landing page is a modern, single destination with all the resources necessary for the different types of personas visiting the website: JSON Schema users, JSON Schema tooling implementers, and other organizations.
+
+The information architecture will make it easier for everyone to find the resources they need and make it easier for the Community to maintain and evolve our docs as the project grows. 
+
+## Who made this possible?
+
+This has been a long journey led by the Community. We want to start by acknowledging all the people who contributed to the previous website versions because this couldn't have been possible without them. THANKS!
```

Comment:
```suggestion
This has been a long journey led by the Community. We want to start by acknowledging all the people who contributed to the previous website versions because this wouldn't have been possible without them. THANKS!
```